pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -1711,6 +1711,129 @@ var _ = Describe("Manager", func() {
 		})
 	})
 
+	Context("other unit tests", func() {
```

Comment:
if you don't think it is worth a specific context, consider just removing the context.

Reply:
Thanks! Done, PTAL.